BrainPortal/public/javascripts/cbrain.js

```diff
@@ -179,7 +179,16 @@
     // Tab Bar, div's of type tabs become tab_bars
     // See TabBar class
     loaded_element.find(".tabs").each( function() {
-      $(this).tabs();
+      $(this).tabs(
```

Comment:
Why is this closing code associated with tags instead of a specific CSS class? Tabs are for the tabbing framework, we're modifying the overlays, use distinct appropriate classes. This code here also means the new overlays cannot be used outside of tabs, for instance.